modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -11,6 +11,8 @@ As a cluster administrator, you can configure an additional network for your VRF
 [NOTE]
 ====
 Applications that use VRFs need to bind to a specific device. The common usage is to use the `SO_BINDTODEVICE` option for a socket. `SO_BINDTODEVICE` binds the socket to a device that is specified in the passed interface name, for example, `eth1`. To use `SO_BINDTODEVICE`, the application must have `CAP_NET_RAW` capabilities.
+
+Using a VRF through the `ip vrf exec` command is not supported in {product-title} pods. To use VRF, applications can bind directly to the VRF interface.
```

Comment:
If there's only one way to do this, I suggest moving the action to the user instead of the application:

```suggestion
Using a VRF through the `ip vrf exec` command is not supported in {product-title} pods. To use VRF, bind applications directly to the VRF interface.
```